live-examples/js-examples/string/string-trim.html

```diff
@@ -0,0 +1,10 @@
+<pre>
+<code id="static-js">var greeting = "   Hello world!   ";
```

Comment:
Shouldn’t we be using `let` or `const` here?

Reply:
Thanks for the review @ExE-Boss , but the style guide says no: https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#language-choice-es5es6